lib/graphql/schema/member.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 require 'graphql/schema/member/accepts_definition'
+require 'graphql/schema/member/base_dsl_methods'
+require 'graphql/schema/member/cached_graphql_definition'
+require 'graphql/schema/member/graphql_type_names'
```

Comment:
All of these were extracted so I could keep `Schema::Member` as a base class, but easily get these methods into `Schema::Interface`.